pkg/kubelet/kubelet.go

```diff
@@ -1319,7 +1313,8 @@ func (kl *Kubelet) initializeModules() error {
 		return fmt.Errorf("Kubelet failed to get node info: %v", err)
 	}
 
-	if err := kl.containerManager.Start(node, kl.GetActivePods, kl.sourcesReady, kl.statusManager, kl.runtimeService); err != nil {
+	if err := kl.containerManager.Start(node, kl.GetActivePods, kl.sourcesReady, kl.statusManager,
+		kl.containerRuntime.(kuberuntime.KubeGenericRuntime).GetRuntimeService()); err != nil {
```

Comment:
is the casting here safe?  It looks like this will panic when using rkt.  You should extend the `kubecontainer.Runtime` interface and implement GetRuntimeService() for rkt.